decidim-core/app/models/decidim/user.rb

```diff
@@ -9,6 +9,8 @@ class User < ApplicationRecord
 
     belongs_to :organization, foreign_key: "decidim_organization_id", class_name: Decidim::Organization
     has_many :authorizations, foreign_key: "decidim_user_id", class_name: Decidim::Authorization, inverse_of: :user
+    has_many :participatory_process_user_roles, foreign_key: "decidim_user_id", class_name: Decidim::ParticipatoryProcessUserRole
```

Comment:
What about extending the `Decidim::User` class with a concern from the `admin` engine?

```ruby
# in admin engine.rb
config.to_prepare do
  Decidim::User.include Decidim::Admin::UserExtends
end

# in an admin concern called user_extends.rb
module Decidim::Admin::UserExtends
  def included(klass)
    klass.extend ClassMethods
  end

  module ClassMethods
    has_many :participatory_process_user_roles, foreign_key: "decidim_user_id", class_name: Decidim::ParticipatoryProcessUserRole
  end
end
```